src/main/java/com/bugsnag/HandledState.java

```diff
@@ -73,6 +99,10 @@ SeverityReasonType calculateSeverityReasonType() {
                 : SeverityReasonType.REASON_CALLBACK_SPECIFIED;
     }
 
+    Map<String, String> getSeverityReasonAttributes() {
+        return severityReasonAttributes;
```

Comment:
We may want to consider wrapping this with `Collections.unmodifiableMap()` to make it very clear that this field shouldn't be mutated.